pkg/kubelet/kubelet.go

```diff
@@ -2388,13 +2388,17 @@ func (kl *Kubelet) isOutOfDisk() bool {
 	// Check disk space once globally and reject or accept all new pods.
 	withinBounds, err := kl.diskSpaceManager.IsRuntimeDiskSpaceAvailable()
 	// Assume enough space in case of errors.
-	if err == nil && !withinBounds {
+	if err != nil {
+		glog.Errorf("failed to check if disk space is available for the runtime: %v", err)
```

Comment:
Yeah. That sounds like a good idea.

On Mon, Jun 6, 2016 at 2:31 PM, Yu-Ju Hong <notifications@github.com> wrote:

> In pkg/kubelet/kubelet.go
> <https://github.com/kubernetes/kubernetes/pull/26910#discussion_r65972965>
> :
>
> > @@ -2388,13 +2388,17 @@ func (kl *Kubelet) isOutOfDisk() bool {
> >  	// Check disk space once globally and reject or accept all new pods.
> >  	withinBounds, err := kl.diskSpaceManager.IsRuntimeDiskSpaceAvailable()
> >  	// Assume enough space in case of errors.
> > -	if err == nil && !withinBounds {
> > +	if err != nil {
> > +		glog.Errorf("failed to check if disk space is available for the runtime: %v", err)
>
> A question beyond this PR: When kubelet cannot check the disk space, it'd
> just admit the pods. That doesn't sound desirable to me... It seems
> reasonable to fail in this case. WDYT?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/26910/files/da8a0f0e3c5e2fa3556078b93ae3ce33d55259c6#r65972965>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AGvIKKFX6vFzCs-JOlznegNGQ9FF4w84ks5qJJGdgaJpZM4IvUy_>
> .
>